specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/HybridCompute.json

```diff
@@ -256,6 +256,214 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HybridCompute/machines/{machineName}/licenseProfile": {
```

Comment:
not sure if you'll get pushback against this path. In other cases, it would follow the form /machines/machineName/licenseProfile/default where we only plan to have one, but I don't know whether that's a hard requirement or not. I'd prefer not to, though I can see some benefit if there is a chance we could add more later.

Reply:
I think it's okay. our /operations /assessPatches and other APIs didn't have the /default either